### PR TITLE
OCPBUGS-1709: Sync javax-mail-api with version required by jenkins 2.346.3

### PR DESCRIPTION
**Pull request by jkhelil**

### CODE REVIEW

**Comment by akram** [comment]
/retitle Sync javax-mail-api with version required by jenkins 2.346.3

File: 2/Dockerfile.localdev [hunk 1]
```diff
@@ -46,16 +46,14 @@ LABEL k8s.io.description="Jenkins is a continuous integration server" \
 # 8080 for main web interface, 50000 for slave agents
 EXPOSE 8080 50000
 
-
+RUN rm /etc/yum.repos.d/*
 COPY contrib/openshift/CentOS-Base.repo /etc/yum.repos.d/CentOS-Base.repo
```
Comment (akram) [inline 2/Dockerfile.localdev:50]: do we still need this?

File: 2/Dockerfile.localdev [hunk 2]
```diff
@@ -46,16 +46,14 @@ LABEL k8s.io.description="Jenkins is a continuous integration server" \
 # 8080 for main web interface, 50000 for slave agents
 EXPOSE 8080 50000
 
-
+RUN rm /etc/yum.repos.d/*
```
Comment (akram) [inline 2/Dockerfile.localdev:49]: maybe group this to reduce the number of layers of the image

File: 2/contrib/openshift/CentOS-Base.repo [hunk 3]
```diff
@@ -9,34 +9,30 @@
...
-[base]
-name=CentOS-7 - Base
-mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=os&infra=$infra
-#baseurl=http://mirror.centos.org/centos/7/os/$basearch/
-gpgcheck=1
-gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+[BaseOS]
+name=CentOS-8 - Base
+mirrorlist=http://mirrorlist.centos.org/?release=8-stream&arch=$basearch&repo=BaseOS&infra=$infra
+gpgcheck=0
```
Comment (akram) [inline 2/contrib/openshift/CentOS-Base.repo:15]: we need to find a way to restore the gpgcheck 
we can probably find the key file somewhere.

**Review by akram — CHANGES_REQUESTED** [review]

File: 2/Dockerfile.localdev (hunk 2 above)
Comment (jkhelil) [reply 2/Dockerfile.localdev:49]: it should be before the copy, we cant group it

File: 2/contrib/openshift/CentOS-Base.repo (hunk 3 above)
Comment (jkhelil) [reply 2/contrib/openshift/CentOS-Base.repo:15]: yes

**Comment by akram** [comment]
/retitle OCPBUGS-1709: Sync javax-mail-api with version required by jenkins 2.346.3

**Pull request merged**